src/hotspot/share/oops/markWord.hpp

```diff
@@ -130,6 +132,7 @@ class markWord {
   static const uintptr_t monitor_value            = 2;
   static const uintptr_t marked_value             = 3;
 
+  static const uintptr_t self_forwarded_value     = 1 << self_forwarded_shift;
```

Comment:
This actually looks like `self_forwarded_in_place`? Because `self_forwarded_value` is just `1`.

Reply:
It's actually self_forwarded_mask_in_place, I removed the _value and simplified the set_self_forwarded() method.

Reply:
Good.